code/modules/mapping/map_template.dm

```diff
@@ -82,6 +82,12 @@
 
 	SSatoms.InitializeAtoms(areas + turfs + movables, returns_created_atoms ? created_atoms : null)
 
+	for(var/turf/unlit as anything in turfs)
+		if(!unlit.always_lit && istype(unlit.loc, /area))
```

Comment:
assuming unlit is always a turf, you can always assume the location is an area

Reply:
Oh yeah I talked with Tivi and forgot to change that I think.